definitions/subannual/seasons.yaml

```
@@ -0,0 +1,5 @@
+# List of seasons
+# the seasons changing dates was chosen here as to minimise the difference of length between seasons as the objective is to compare energy amounts per seasons
+
+- {Season Input}:
```

You have to use quotes if the first character is a special character like `{`...
```suggestion
- "{Season Input}":
```